airflow/www/templates/appbuilder/baselayout.html

```diff
@@ -75,15 +75,46 @@
 
 
 {% block tail_js %}
-{{ super() }}
-<script type="text/javascript">
-  // below variables are used in base.js
-  var hostName = '{{ hostname }}';
-  var csrfToken = '{{ csrf_token() }}';
-  $("time[title]").tooltip()
-</script>
-<!--[if IE ]>
-<script src="{{ url_for_asset('ie.js') }}" type="text/javascript"></script>
-<![endif]-->
-<script src="{{ url_for_asset('base.js') }}" type="text/javascript"></script>
+  {{ super() }}
+
+  <script type="text/javascript">
+    // below variables are used in base.js
+    var hostName = '{{ hostname }}';
+    var csrfToken = '{{ csrf_token() }}';
+    $("time[title]").tooltip()
+  </script>
+  <!--[if IE ]>
+  <script src="{{ url_for_asset('ie.js') }}" type="text/javascript"></script>
+  <![endif]-->
+  <script src="{{ url_for_asset('base.js') }}" type="text/javascript"></script>
+
+  {% if analytics_tool == 'google_analytics' %}
```

Comment:
Maybe to make this a bit easier to manger we could do:

```
       {% include "analytics_" + analytics_tool + "-partial.html" %}
```

And then have `analytics_google_analytics-partial.html`, `analytics_segmet-partial.html` etc.

Reply:
good idea, on it